internal/presenters/presenter_local_finding.go

```diff
@@ -27,27 +27,16 @@ var embeddedFiles embed.FS
 type TemplateImplFunction func() (*template.Template, template.FuncMap, error)
 
 type LocalFindingPresenter struct {
-	ShowIgnored      bool
-	Input            *local_models.LocalFinding
-	OrgName          string
-	TestPath         string
-	SeverityMinLevel string
-	config           configuration.Configuration
-	writer           io.Writer
-	templateImpl     map[string]TemplateImplFunction
-}
-
-type PresentationFindingResource struct {
-	local_models.FindingResource
-}
-
-func (pfr *PresentationFindingResource) IsIgnored() bool {
-	return pfr.Attributes.Suppression != nil
+	TestPath     string
+	Input        *local_models.LocalFinding
+	config       configuration.Configuration
+	writer       io.Writer
+	templateImpl map[string]TemplateImplFunction
 }
 
 type FilteredFindings struct {
-	OpenFindings    []*PresentationFindingResource
-	IgnoredFindings []*PresentationFindingResource
+	OpenFindings    []local_models.FindingResource
```

Comment:
Question: should we just have all findings and methods injected to filter by state?